orca/orca-integrations-gremlin/src/main/java/com/netflix/spinnaker/orca/config/GremlinConfiguration.kt

```diff
@@ -24,34 +20,22 @@ import retrofit.client.Client
 )
 class GremlinConfiguration {
 
-  @Bean
-  fun gremlinEndpoint(
-    @Value("\${integrations.gremlin.base-url:https://api.gremlin.com/v1}") gremlinBaseUrl: String
-  ): Endpoint {
-    return Endpoints.newFixedEndpoint(gremlinBaseUrl)
-  }
-
   @Bean
   fun gremlinService(
-    retrofitClient: Client,
-    gremlinEndpoint: Endpoint,
-    spinnakerRequestInterceptor: RequestInterceptor
+    @Value("\${integrations.gremlin.base-url:https://api.gremlin.com/v1}") gremlinBaseUrl: String,
+    serviceClientProvider: ServiceClientProvider
   ): GremlinService {
     val mapper = OrcaObjectMapper
       .newInstance()
       .setPropertyNamingStrategy(
         PropertyNamingStrategy.SNAKE_CASE
       )
       .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS) // we want Instant serialized as ISO string
-    return RestAdapter.Builder()
-      .setRequestInterceptor(spinnakerRequestInterceptor)
-      .setEndpoint(gremlinEndpoint)
-      .setClient(retrofitClient)
-      .setLogLevel(RestAdapter.LogLevel.BASIC)
-      .setErrorHandler(SpinnakerRetrofitErrorHandler.getInstance())
-      .setLog(RetrofitSlf4jLog(GremlinService::class.java))
-      .setConverter(GremlinConverter(mapper))
```

Comment:
Don't we still need GremlinConverter, or something like it?  If we really don't, let's delete GremlinConverter.java.

Reply:
I was a miss. I will add the converter.

Reply:
Deleted GremlinConverter.